### PR TITLE
explain use case for Server Key auth & remove JWT

### PR DESCRIPTION
**Pull request by bradoyler**
To clear up confusion around Auth strategies

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -157,6 +157,7 @@ Youtube.authenticate({
 ```
 
 #### Server Key
+*only for request that don't require [user authorization](https://developers.google.com/youtube/v3/guides/authentication) (certain list operations)*
```
Comment (IonicaBizau) [inline README.md]: See [these comments](https://github.com/IonicaBizau/youtube-api/pull/40/files#r38647251)–btw, you can [edit this in this branch](https://github.com/bradoyler/youtube-api/edit/patch-2/README.md) and the pull request will be updated. :four_leaf_clover:

**Comment by bradoyler** [comment]
@IonicaBizau thx. 1 more commit coming soon.

**Comment by IonicaBizau** [comment]
Looks good! [Add yourself as contributor](https://github.com/IonicaBizau/youtube-api/blob/master/CONTRIBUTING.md#fixing-issues), and then I will merge it! :+1:

**Comment by IonicaBizau** [comment]
Thanks! :sparkles: 

:ship: `1.1.0` :tada:

**Comment by bradoyler** [comment]
I think expanding the examples may also be a worthwhile contribution. 
Thoughts?

**Comment by IonicaBizau** [comment]
Sure! Contributions are more than welcome! :smile: I will probably comment if I see something that can be improved, but that doesn't mean rejection. :smile:

**Pull request merged**